src/View/Admin/WooTab.php

```diff
@@ -250,7 +250,7 @@ private function get_filtered_settings( array $settings ) {
 	private function get_default_settings_options() {
 		$settings = [];
 
-		switch ( $GLOBALS['current_section'] ) {
+		switch ( $GLOBALS['current_section'] ?? '' ) {
```

Comment:
I think this one would just fall down into the empty string an default already, but I could be wrong.

Reply:
@tw2113 similar to the previous instance, since we're not actually checking if that key exists, it would still trigger the warning if it's not set.